ctapipe-extra

```diff
@@ -1 +1 @@
-Subproject commit f7fe2209646dc8f99a28ec9ab4df8257687e1896
+Subproject commit 536490d27e34c5f3b0c361f828de208f7e17b60f
```

Comment:
You should also avoid doing 
```
git add .
```

It is less error-prone to just add the files you want to commit explicitly.
Also helpful is `git status --short`, which only shows an overview over the files.